### DISCUSSION TITLE
Received errors while using Safe Exam Browser on MAC version 3.2.5

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-mac#312] [Q&A]
Hi Team,

We have installed SEB for MAC version 3.2.5 on laptop and tried to attend the exam using SEB file which we have configured. But, In the middle of the exam we have received "Load Error".

Laptop OS: "MAC OS 13 Ventura"
SEB Version: "3.2.5"

Here I have attached the error screenshots for your reference.

Kindly check and let us know the reason for this error & what we need to change/configure to fix this.
![IMG_3648](https://github.com/SafeExamBrowser/seb-mac/assets/111885140/8bd18abe-66d5-4dbf-850f-2e3187421113)
![IMG_3649](https://github.com/SafeExamBrowser/seb-mac/assets/111885140/834a93e7-26b7-4f94-b4bc-ac125d122eed)

**Comment by danschlet** [comment]
That's no known error. "Load Error" is displayed for any kind of errors the macOS browser and networking engine encounters when your web content interacts with the browser, the rest of the error message comes directly from these system components. You should ask the developer of your web application what the reason for this error could be, as it seems quite specific. "FetchEvent.respondWith" is some web API, if you do a [web search](https://www.google.com/search?client=safari&rls=en&q=%22FetchEvent.respondWith%22&ie=UTF-8&oe=UTF-8) you get [results like this](https://github.com/mswjs/msw/issues/558) or [this](https://stackoverflow.com/questions/43956230/a-serviceworker-passed-a-promise-to-fetchevent-respondwith-that-resolved-with), which seems to suggest that it is a compatibilitly issue of Safari (therefore the WebKit browser engine, which SEB also uses), which maybe can be solved in your web application.

SEB can only change few very high level settings of the system's browser engine, which it uses, so mostly you can't do anything with SEB settings in such cases. And in any case if we could do anything to analyze such an issue, we would always at least need the SEB log files and the configuration file you used. So please always provide those and the information listed on our [support page](https://stackoverflow.com/questions/43956230/a-serviceworker-passed-a-promise-to-fetchevent-respondwith-that-resolved-with).
